src/nv_ingest/extraction_workflows/pdf/pdfium_helper.py

```diff
@@ -53,6 +55,7 @@
 
 
 logger = logging.getLogger(__name__)
+configure_logging(logger, os.getenv("INGEST_LOG_LEVEL", "INFO"))
```

Comment:
This shouldn't be required, logging should be globally configured here: https://github.com/NVIDIA/nv-ingest/pull/14/files#diff-a5b564197d741ea776ee29ab7192991961c613741edc265c9c623c58c0a623cdL657